packages/luthor/lib/src/validations/doubles/max_validation.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:luthor/src/validation.dart';
+
+class DoubleMaxValidation extends Validation {
+  double maxValue;
+  String? customMessage;
+
+  DoubleMaxValidation({
+    required this.maxValue,
+    String? message,
+  }) : customMessage = message;
+
+  @override
+  bool call(String? fieldName, Object? value) {
+    super.call(fieldName, value);
+
+    if (value == null) return true;
+    if (value is! double) return false;
```

Comment:
Maybe have it just check for `num` since the prevuious `.double()` will already validate this
```suggestion
    if (value is! num) return false;
```